lib/reports/ppl-applications/index.js

```diff
@@ -52,7 +55,7 @@ module.exports = ({ db, query: params, flow }) => {
         timers.licensing += diff;
       }
       timers.total += diff;
-      last = moment(record.created_at).valueOf();
+      last = moment(log.created_at).valueOf();
```

Comment:
This is the key bit. It wasn't actually updating before, so we were summing the difference between each activity and the task creation.